v22.1/online-schema-changes.md

```diff
@@ -42,6 +42,17 @@ Once backfilling is complete, all nodes will switch over to the new schema, and
 
 For more technical details, see [How online schema changes are possible in CockroachDB][blog].
 
+{% include_cached new-in.html version="v22.1" %} Online schema changes that trigger an index backfill (adding data to an index) will now pause if the node executing the schema change is running out of disk space. The following statements will now pause if the node executing the schema change is running out of disk space:
+
+- [`ADD COLUMN`](add-column.html)
+- [`ALTER PRIMARY KEY`](alter-primary-key.html)
+- [`CREATE INDEX`](create-index.html)
+- [`CREATE TABLE AS`](create-table-as.html)
+- [`REFRESH`](refresh.html)
```

Comment:
Is the omission of `CREATE MATERIALIZED VIEW` deliberate?

Reply:
It was an accident. Including it now.